azure/template.json

```diff
@@ -0,0 +1,193 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
You'll want `2018-05-01/subscriptionDeploymentTemplate.json#` for subscription ARM template deployments